src/main/resources/templates/nonMedicalUser-Home.html

```diff
@@ -3,16 +3,19 @@
 
 <head>
   <title>Peace Of Mind</title>
+  <link href="/css/styles.css" rel="stylesheet">
 </head>
 
 <body class="container">
 
   <header>
     <h1>Peace of Mind</h1>
     <nav>
-      <h2>
         <a th:href="@{/non-medical-user(id=${nonMedicalUser.id})}">Account</a>
-      </h2>
+          <a th:href="@{/non-medical-user-home(id=${nonMedicalUser.id})}">Home</a>
```

Comment:
Let's remove these please, as they all point to the homepage, which the user is already on